cmd/kuberhealthy/reaper.go

```diff
@@ -40,8 +44,26 @@ func init() {
 		cfg.MaxCheckPods = maxCheckerPodsDefault
 	}
 
-	if cfg.JobCleanupDuration == 0 {
-		cfg.JobCleanupDuration = jobCleanupDurationDefault
+}
+
+// parseDuration parses checkReaper duration configs into time.Duration. Go parses time.Duration from int64 and not from
+// string, which is how the config reloader unmarshals the config json. This way we can still supply time.Duration type
+// values in the config, instead of an int64 that doesn't allow users to specify the duration type.
+func parseDuration() {
```

Comment:
Consider making this a method on the `Config` struct.  This currently sits in the global namespace for the binary.